CHANGELOG.md

```diff
@@ -3,7 +3,33 @@
 ## Unreleased
 
 ### Breaking changes
-You must make the following changes when you migrate to this release, or your service may break.
+Where applicable, you must make the following changes when you migrate to this release, or your service may break.
+
+#### Check your accordions still display as expected
+
+[See the new design and revised guidance for accordions in the Design System](https://design-system.service.gov.uk/components/accordion/). In particular, review the guidance on when and when not to use accordions.
+
+You should check that your accordions still display as expected if you:
+
+- have applied any custom styles to your accordions
+- use accordions in non-standard ways
+- have section headings that contain unusual formatting (for example, wrappers)
+
+You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
+
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
+
+#### Review and revise the accordion's summary line content
+
+If you're already using the summary line, review your use to make sure you actually need it.
+
+If you need to keep the summary line, [review the accordion guidance](https://design-system.service.gov.uk/components/accordion/) to make it as short as possible.
```

Comment:
Should we link to the live guidance equivalent of https://deploy-preview-1961--govuk-design-system-preview.netlify.app/components/accordion/#adding-a-summary-line to get them to the relevant bit in the guidance?

Reply:
Yeah, Calvin's rationale sounds good.